inference-engine/tests/unit/vpu/middleend_tests/passes_tests/adjust_data_batch_tests.cpp

```diff
@@ -0,0 +1,389 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vpu/middleend/allocator/allocator.hpp>
+#include <vpu/stages/mx_stage.hpp>
+#include <vpu/utils/numeric.hpp>
+
+#include "graph_transformer_tests.hpp"
+
+using namespace vpu;
+
+class VPU_AdjustDataBatchTest : public GraphTransformerTest {
+public:
+    int batchSize;
+
+    TestModel testModel;
+
+    void SetUp() override {
+        ASSERT_NO_FATAL_FAILURE(GraphTransformerTest::SetUp());
+
+        ASSERT_NO_FATAL_FAILURE(InitCompileEnv());
+
+        batchSize = 4;
```

Comment:
It's actually a constant now, not a variable, so I see two options there: make `batchSize` const or try to parametrize the cases with batchSize number. I prefer the second.

Reply:
Actually the idea of my comment was not to hardcode this value in all places, but just to declare it as `constexpr int`.

Reply:
It causes failures. I think I will choose the first option.